packages/contracts/src/contracts/current/multisig/MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress/MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress.sol

```diff
@@ -18,15 +18,19 @@
 
 pragma solidity ^0.4.10;
```

Comment:
Since we'll have to redeploy this with the new functionality should we bump the solidity version up?

Reply:
Hmm, the base MultiSig contracts have been heavily audited with 0.4.10, so it might not be worth bumping up. We might have to downgrade some of our syntax if we want to keep the old version though.